### ISSUE TITLE
[accessibility] Attribution container should not have `role="list"` (or should have `listitem` children)

### ISSUE THREAD

**Issue by Malvoz** [mapbox/mapbox-gl-js#11033]
https://github.com/mapbox/mapbox-gl-js/blob/36533f3683dd48837ab07095fb6484165d9ee400/src/ui/control/attribution_control.js#L64

The attribution links should either be children of list items ([`role="listitem"`](https://www.w3.org/TR/wai-aria/#listitem)) or [`role="list"`](https://www.w3.org/TR/wai-aria/#list) should be removed.

See motivation in https://dequeuniversity.com/rules/axe/4.2/list?application=RuleDescription (describes `<ul>` and `<li>` requirements, the same is applicable to the equivalent ARIA roles `list` and `listitem`).

**Comment by avpeery** [comment]
Hi @Malvoz, thanks so much for your time! It seems to me that this specific accessibility issue is set up correctly. Please correct me if I am misunderstanding. The role='list' attribute is set on the attribution container as a whole. When I go to inspect (on Chrome and Safari), the attribution links have the role listitem. Here are two screenshots from inspecting the attribution container element.
<img width="331" alt="Screen Shot 2021-10-12 at 10 11 19 AM" src="https://user-images.githubusercontent.com/42715836/137000418-fbc7d63f-c1b9-4d59-a39e-4c6c5e71ecdd.png">
.
<img width="873" alt="Screen Shot 2021-10-12 at 10 10 13 AM" src="https://user-images.githubusercontent.com/42715836/137000428-919f132f-7f07-4260-8a4c-dee5f31ef7b5.png">

**Comment by Catalyst497** [comment]
I'm interested in solving the little issue here. Could you assign it to me?
Please what file(s) am I going to be looking for here? 

I am a first timer by the way.

**Comment by mrwweb** [comment]
It would be great to get this resolved! Noted accessibility expert @marcysutton caught this issue in a [recent video review](https://testingaccessibility.com/accessibility-reviews/wta) of a site using Mapbox.

As @Malvoz notes, overriding the default semantics of a link is a bad idea, and it doesn't seem like these really need to be in a list. (If they did, you could also put them in a real list!)

Looking at [this Mapbox demo](https://api.mapbox.com/styles/v1/mapbox/outdoors-v12.html?title=true&access_token=pk.eyJ1IjoibWFwYm94IiwiYSI6ImNpejY4M29iazA2Z2gycXA4N2pmbDZmangifQ.-g_vE53SD2WrJ6tFX7QHmA#7.5/42.2/9.1), I could not find anything with `role="list"`, so I'd recommend just removing `role="listitem"`.

If @Catalyst497 is still willing, maybe someone could assign them this task and fix this!?

**Comment by tristen** [comment]
> Would you be able to handle this?

👋 @kumiko-haraguchi Thanks for the ping. I can handle those attributes in the TileJSON documents. Want to simplify that draft PR and assign a review from me?

**Comment by kumiko-haraguchi** [comment]
Hi @tristen 👋
Thank you so much for your help! Unfortunately, I don't seem to have permission to add a reviewer for my PR. But please feel free to edit it🙏 

![Screenshot 2023-08-19 at 11 44 16](https://github.com/mapbox/mapbox-gl-js/assets/28984604/fc20048f-ee78-4f11-bc23-97bccee318f2)

**Comment by Malvoz** [comment]
Fixed in https://github.com/mapbox/mapbox-gl-js/pull/12857.
